### PR TITLE
Update some scenario natives

### PR DESCRIPTION
**Pull request by coalaura**
Updated various scenario natives.

### CODE REVIEW

File: PED/IsPedUsingAnyScenario.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ ns: PED
 BOOL IS_PED_USING_ANY_SCENARIO(Ped ped);
 ```
 
+Returns true if the ped is using any scenario.
```
Comment (AvarianKnight) [inline PED/IsPedUsingAnyScenario.md:11]: This should be under `## Return value`

File: PED/IsPedUsingScenario.md [hunk 2]
```diff
@@ -8,9 +8,10 @@ ns: PED
 BOOL IS_PED_USING_SCENARIO(Ped ped, char* scenario);
 ```
 
+Returns true if the specified ped is using the specified scenario. See [`TASK_START_SCENARIO_IN_PLACE`](#_0x142A02425FF02BD9) for a list of scenarios.
```
Comment (AvarianKnight) [inline PED/IsPedUsingScenario.md:11]: This should be under `## Return value`

File: TASK/IsPedActiveInScenario.md [hunk 3]
```diff
@@ -8,9 +8,9 @@ ns: TASK
 BOOL IS_PED_ACTIVE_IN_SCENARIO(Ped ped);
 ```
 
-When passing a ped parameter, the function returns true if the ped is currently in any scenario.
+When passing a ped parameter, the function returns true if the ped is currently in any scenario. The ped is really only "in" the scenario when they are playing the ambient animations associated with it, unlike [`IS_PED_USING_ANY_SCENARIO`](#_0x57AB4A3080F85143).
```
Comment (AvarianKnight) [inline TASK/IsPedActiveInScenario.md:11]: The current description and new description kinda counter eachother 
```suggestion
This is a stricter version of [`IS_PED_USING_ANY_SCENARIO`](#_0x57AB4A3080F85143)
```

File: TASK/IsPedActiveInScenario.md [hunk 4]
```diff
@@ -8,9 +8,9 @@ ns: TASK
...
 
 ## Parameters
-* **ped**: 
+* **ped**: The ped handle.
 
 ## Return value
```
Comment (AvarianKnight) [inline TASK/IsPedActiveInScenario.md:16]:
```suggestion
## Return value
Returns true if the ped is in a using a scenario and is doing the ambient animations.
```

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: PED/IsPedUsingAnyScenario.md (hunk 1 above)
Comment (coalaura) [reply PED/IsPedUsingAnyScenario.md]: fixed

File: PED/IsPedUsingScenario.md (hunk 2 above)
Comment (coalaura) [reply PED/IsPedUsingScenario.md]: fixed

**Pull request merged**